src/test/scala/ie/zalando/fabric/gateway/service/IngressDerivationChainSpec.scala

```diff
@@ -715,6 +715,21 @@ class IngressDerivationChainSpec extends FlatSpec with MockitoSugar with Matcher
     }
   }
 
+  "Access Logging" should "add the unverifiedAuditLog filter to all service routes with the sub key" in {
+    val ingresses = testableWhitelistRoutesDerivation(sampleGateway)
+
+    val filteredRoutes = ingresses
+      .filterNot(isAdminRoute)
+      .filterNot(isCatchAllRoute)
```

Comment:
Should it be added to the catch all route as well? After the token check I mean

Reply:
Catch all routes are shunts though. I didnt even check if they appear in the access log

Reply:
Ok, did a test ingress like below.
Looks like it works.
Will update, although I can't target users then. But we'll just default to assuming more service traffic than user traffic

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    zalando.org/aws-load-balancer-ssl-policy: ELBSecurityPolicy-FS-2018-06
    zalando.org/skipper-routes: Path("/rejected") && Method("GET") && JWTPayloadAnyKV("https://identity.zalando.com/managed-id", "bmooney")
      -> status(400) -> unverifiedAuditLog() -> inlineContent("{\"title\":\"LOGGED REJECTION\",\"status\":400,\"detail\":\"TLS
      is required\",\"type\":\"https://cloud.docs.zalando.net/howtos/ingress/#redirect-http-to-https\"}")
      -> <shunt>
  name: brian-standalone-test-shunt
  namespace: fabric
spec:
  rules:
  - host: demo-gw-ss-integration.smart-product-platform-test.zalan.do
    http:
      paths:
      - backend:
          serviceName: demo-gw-ss-integration-v1
          servicePort: 5678
```

Reply:
They do appear in access logs, checked there

Reply:
Should remove .filterNot(isCatchAllRoute) from the test as well

Reply:
custom routes are modelled slightly differently. They're covered in a diff test. Apologies, forgot to push on Friday...